contents/blog/plugins-part-1-the-world-map.md

```diff
@@ -0,0 +1,106 @@
+---
+date: 2021-09-09
+title: "Plugging in the Plugins. Part 1: The World Map"
```

Comment:
I don't think "The World Map" is the right title, since that's what sparked this, but it completely doesn't get delivered. I think for an HN-like audience a title we should somehow convey that this is about building plugins from scratch.

Reply:
"How We Added Plugin Support To Our Analytics Tool, Pt. 1: The World Map"

^